configs/mmdet/_base_/base_openvino_dynamic.py

```diff
@@ -0,0 +1,6 @@
+_base_ = ['./base_dynamic.py', '../../_base_/backends/openvino.py']
+
+onnx_config = dict(input_shape=None)
+
+backend_config = dict(
+    model_inputs=[dict(opt_shapes=dict(input=[1, 3, 300, 300]))])
```

Comment:
The file name should contain the opt shape info. Please append `300x300` to the file name.